doc/tutorial/read_mapper/read_mapper_step3.cpp

```diff
@@ -58,11 +58,10 @@ void map_reads(std::filesystem::path const & query_path,
 
     for (auto & [query, id, qual] : query_in | seqan3::views::take(20))
     {
-        auto positions = search(query, index, search_config);
-        for (auto & [idx, pos] : positions)
+        for (auto & [query_idx, text_pos] : search(query, index, search_config))
         {
-            size_t start = pos ? pos - 1 : 0;
-            std::span text_view{std::data(storage.seqs[idx]) + start, query.size() + 1};
+            size_t start = text_pos.second ? text_pos.second - 1 : 0;
```

Comment:
Huh, so the result now is `{id, {id, pos}}`? I thought the result range is flattened.

Reply:
The result range is flattened. But example does process the queries query by query. In this case we get the id of the query and the position within the reference index which is a collection. Hence, the return type as a pair. This will be modified once this is merged with the search result